source/common/res/features/budget-quick-switch/main.js

```diff
@@ -0,0 +1,42 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
+    ynabToolKit.activityTransactionLink = (function () {
+      let applicationController = ynabToolKit.shared.containerLookup('controller:application');
+      let userBudgetsController = ynabToolKit.shared.containerLookup('controller:users/budgets');
+
+      function populateBudgetList() {
+        let budgets = userBudgetsController.get('budgets');
+        let currentBudgetId = applicationController.get('activeBudgetVersion').get('entityId');
+        let $openBudgetListItem = $('.modal-select-budget').find('.modal-select-budget-open').parent();
+
+        budgets.forEach((budget) => {
+          if (budget.get('budgetVersionId') === currentBudgetId) return;
+
+          let budgetListItem = $(`<li>
+                                    <button>
+                                      <i class="flaticon stroke mail-1"></i>
+                                      ${budget.get('budgetVersionName')}
```

Comment:
Theoretically this could cause DOM injection, if the name of a budget was '<script>alert("do evil thing.");</script>' yeah?

If so, this was flagged during Firefox review, and we ended up switching everything to the JQuery style of DOM creation like this:

```javascript
            $('.nav-main').append(
              $('<li>').append(
                $('<li>', { class: 'ember-view navlink-reports' }).append(
                  $('<a>', { href: '#' }).append(
                    $('<span>', { class: 'ember-view flaticon stroke document-4' })
                  ).append(
                    (ynabToolKit.l10nData && ynabToolKit.l10nData['sidebar.reports']) || 'Reports'
                  )
                )
              )
            );
```

So if I'm right, please change it to the JQuery style, and if I'm not (I'm not particularly familiar with template literals, and I'd actually much prefer to use this style throughout the entire code base!) could you point me to some docs that confirm it so I have evidence if Mozilla knock us back?

Thanks!